src/Connection/Connection.php

```diff
@@ -372,6 +373,11 @@ public function areSlavesEnabled(): bool
         return $this->enableSlaves;
     }
 
+    public function isAutoSlaveForReadQueriesEnabled(): bool
```

Comment:
Please add method to interface

Reply:
There are many another methods, that used in `Command` class, but not added to `ConnectionInterface`
Is this made by design or by mistake?